py-scripts/lf_interop_qos.py

```diff
@@ -393,7 +393,7 @@ def phantom_check(self):
 
         response_port = self.json_get("/port/all")
 
-        mac_id1_list = []
+        
```

Comment:
If removing this line, please remove space as well (should only be one line of space unless splitting functions outside a class).